presto-tpcds/src/main/resources/META-INF.services/com.facebook.presto.spi.Plugin

```diff
@@ -0,0 +1 @@
+com.facebook.presto.tpcds.TpcdsPlugin
```

Comment:
This is part of `presto-maven-plugin` (https://github.com/prestodb/presto-maven-plugin).
This manifest is created by that plugin for all the other connectors.